sqlmesh/core/engine_adapter/bigquery.py

```diff
@@ -469,8 +470,55 @@ def _create_table_properties(
         if clustered_by:
             properties.append(exp.Cluster(expressions=[exp.column(col) for col in clustered_by]))
 
+        if table_description:
+            properties.append(
+                exp.Property(
+                    this=exp.Identifier(this="description", quoted=False),
+                    value=exp.Literal.string(table_description),
+                )
+            )
+
         for key, value in (table_properties or {}).items():
-            properties.append(exp.Property(this=key, value=value))
+            properties.append(exp.Property(this=key, value=value.copy()))
+
+        if properties:
+            return exp.Properties(expressions=properties)
+        return None
+
+    def _build_col_comment_exp(
+        self, col_name: str, column_descriptions: t.Dict[str, str]
+    ) -> t.List[exp.ColumnConstraint | None]:
+        comment = column_descriptions.get(col_name, None)
+        if comment:
+            return [
+                exp.ColumnConstraint(
+                    kind=exp.Properties(
+                        expressions=[
+                            exp.Property(
+                                this=exp.Identifier(this="description", quoted=False),
```

Comment:
Ditto about the description. Should we just factor this part out into a helper method?